### DISCUSSION TITLE
Did is possible to someone show a instalation method?

### DISCUSSION THREAD

**Discussion by Vectruz** [ChanpleCai/SmartTaskbar#137] [Ideas]
I mean, the project is nice, but how exactly people get in hads to use it? you download source code and take on some places of windows, or has a install file, or what exactly need to be done to run this project on a computer?

**Comment by zero-stroke** [comment]
No idea

**Comment by kmccb** [comment]
I have the same question. How do you install it from the github download? Thanks

**Comment by chabu-dai** [comment]
https://github.com/Oliviaophia/SmartTaskbar/releases
